.github/workflows/website.yml

```diff
@@ -0,0 +1,25 @@
+name: website
+on:
+  push:
+    branches: [main]
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pages: write
+      id-token: write
+    environment:
+      name: Website
+      url: ${{ steps.deployment.outputs.page_url }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v3
+        with: {node-version: 18}
```

Comment:
```suggestion
        with:
          node-version: 18
```

Reply:
Why 18? `node` is the always stable one. Perhaps that?

Reply:
I copy-pasted this and forgot to update.